### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by alejandroereyes**
Refactored and corrected create, update, and destroy room actions. 
Added admin nav to app html, only shows if admin user logged in is admin.
Only admin can create, update, and destroy a room. 
Admin dashboard is set up.

### CODE REVIEW

File: app/models/meeting.rb [hunk 1]
```diff
@@ -2,4 +2,18 @@ class Meeting < ActiveRecord::Base
...
+  #   meetings.each do |meeting| # loop thru meetings
+
+  #    if meeting.private # check if meeting is private
+  #     @all_meetings << meeting if current_user.id == meeting.employee.id || user_is_admin?
+  #    else
+  #     @all_meetings << meeting
+  #    end
+  #   end
+  #   @all_meetings # send to display
+  # end
```
Comment (jah2488) [inline app/models/meeting.rb]: So many comments

**Comment by BenHand** [comment]
:+1:

**Comment by jah2488** [comment]
:+1:

**Comment by jah2488** [comment]
@alejandroereyes Can't be automerged anymore, gotta pull down and conflict manage.

**Comment by alejandroereyes** [comment]
resolved merge conflicts

**Comment by JMD1986** [comment]
:+1:

**Comment by BenHand** [comment]
:+1:

**Pull request merged**